pb_plugins/dcsdkgen/type_info.py

```diff
@@ -61,3 +77,9 @@ def is_primitive(self):
         """ Check if the field type is primitive (e.g. bool,
         float) or not (e.g message, enum) """
         return self._field.type not in {11, 14}
+
+    @property
+    def is_repeated(self):
+        """ Check if the field is a repeated type (in which
+        case it is converted to a list) """
+        return self._field.label == 3
```

Comment:
Shouldn't this number 3 be in some define list?

Reply:
+1, I also think this should be defined somewhere

Reply:
:+1: 

I'll check how I can do that!